runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1247,6 +1247,19 @@ gcParseXXArguments(J9JavaVM *vm)
 		}
 	}
 
+	{
+		IDATA portableSharedCacheIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, VMOPT_XXPORTABLESHAREDCACHE, NULL);
+		IDATA noportableSharedCacheIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, VMOPT_XXNOPORTABLESHAREDCACHE, NULL);
+		if (portableSharedCacheIndex != noportableSharedCacheIndex) {
+			/* At least one option is set. Find the right most one. */
+			if (portableSharedCacheIndex > noportableSharedCacheIndex) {
+				extensions->portableSharedCacheEnabled = true;
+			}
+			else {
```

Comment:
please join two lines to one:
```
    } else {
```

Reply:
oops, didn't realize this was the format problem you were talking about, now i get it, thanks!